products/extensions/search/completion.md

```diff
@@ -1,16 +1,16 @@
 # Completion
 
-The Advanced Search does not use the default [Elasticsearch Completion](https://www.elastic.co/guide/en/elasticsearch/reference/7.10/search-suggesters.html#completion-suggester), because it does only support a fixed order and the storage size is high. As an alternative, the Advanced Search uses aggregations to find the most important words combinations for your search input.
+The *advanced search* does not use the default [Elasticsearch Completion](https://www.elastic.co/guide/en/elasticsearch/reference/7.10/search-suggesters.html#completion-suggester) because it does only support a fixed order and the storage size is high. As an alternative, *advanced search* uses aggregations to find the most important word combinations for your search input.
 
-The Full Text Boosted field is used to generate a list of completions. Each word is a separate completion suggestion.
+The `FullText` Boosted field is used to generate a list of completions. Each word is a separate completion suggestion.
 
 ## Extension for compound completions
 
 {% hint style="warning" %}
-The default Advanced Search does not support compound completions from multiple words.
+The default *advanced search* does not support compound completions from multiple words.
```

Comment:
Advanced Search is the name of the feature, so it should be uppercase everywhere - also the following/previous mentions.